sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
sensor_msgs/CameraInfo cam_info
geometry_msgs/PointStamped location
---
geometry_msgs/PointStamped[] slot_locations
